00_intro.Rmd

```diff
@@ -1,66 +0,0 @@
-{
-course-completeness: 100
-course-attempts: 2
-default-quiz-attempts: 2
-default-random-choice-order: true
-default-quiz-show-answers: none
-}
-
-# Where Does Data Come From?
```

Comment:
This seemed like either it should be in the very beginning or we don't need it. I opted for that we don't need it.